solidity/contracts/LPRewards.sol

```diff
@@ -236,3 +236,24 @@ contract LPRewardsKEEPTBTC is LPRewards {
         LPRewards(keepToken, keepTbtcUniswapPair)
     {}
 }
+
+/// @title KEEP rewards for the tBTC Saddle liquidity pool.
+contract LPRewardsTBTCSaddle is LPRewards {
+    bool public _gated = true;
+
+    constructor(KeepToken keepToken, IERC20 tbtcSaddleLPToken)
+        public
+        LPRewards(keepToken, tbtcSaddleLPToken)
+    {}
+
+    // if the pool is gated, disallow tx's that didn't come from msg.sender.
+    function stake(uint256 amount) public {
+        // solium-disable-next-line security/no-tx-origin
+        require(!_gated || msg.sender == tx.origin, "EOA only plz");
```

Comment:
What does EOA mean? Can we improve the error message?

Reply:
Externally Owned Account, not a contract.

Reply:
Thanks! So maybe we could change `EOA` to `Externally Owned Account`?

Reply:
Sure, updated in https://github.com/keep-network/keep-ecdsa/pull/675/commits/32e500506ac1f7c64e72c1b77eff13076ff5ba30.